dto/src/main/java/org/jboss/pnc/dto/requests/validation/VersionValidationRequest.java

```diff
@@ -0,0 +1,21 @@
+package org.jboss.pnc.dto.requests.validation;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import lombok.Builder;
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+@Builder(builderClassName = "Builder")
+@JsonDeserialize(builder = VersionValidationRequest.Builder.class)
+public class VersionValidationRequest {
+
+    @NotBlank
+    public final String version;
```

Comment:
I think the final decision was to use "value" attribute?

Reply:
I've discussed with @janinko whether to have a RequestDTO that is done in a general way and takes f.e. one string. It was decided that it could cause confusion because some DTO's could take a List of strings, number or List of numbers. For each, there would have to be a separate class with names like StringRequest, StringListRequest... and in the code, it could potentially affect readability. To be sure we went with concrete naming and attributes.

Reply:
As it is `/product-milestones/validate-version` it make sense to me that the field is named `version`. What's your reasonto nameit `value`, @matedo1?

Reply:
my first proposal was to name it `version`, we considered making it more generic while using `value` (as it could be reused for other string validations too), but as API consumer I totally agree also with `version` that is actually slightly better and easier to read, so if it's ok for you guys, let's use it